src/Format.psm1

```diff
@@ -10,7 +10,7 @@ function Format-Collection ($Value, [switch]$Pretty) {
     }
     $count = $Value.Count
     $trimmed = $count  -gt $Limit
-    '@(' + (($Value | Select -First $Limit | % { Format-Nicely -Value $_ -Pretty:$Pretty }) -join $separator) + $(if ($trimmed) {', ...'}) + ')'
```

Comment:
Can we guarantee that `$Value` is not an empty collection? `IndexOutOfRangeException` on `@()[0..0]` in strictmode. Same in `Sort-Property`

Reply:
not this way. I did not think about that. 👍